syft/frameworks/torch/tensors/interpreters/utils.py

```diff
@@ -23,3 +30,55 @@ def hook_args(self, *args, **kwargs):
         return response
 
     return hook_args
+
+
+overloaded.method = overload_method
+
+
+def overload_function(attr):
```

Comment:
I think this file should probably move up a few directories. It doesnt really belong in interpreters and should probably just be in the same directory as the hook

Reply:
Yes I agree! Also it should also be in a normal class i got confused using the Object()